src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
+  <div data-qa="notification" class="success">${message}</div>
+  `);
+};
+
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
+  <div data-qa="notification" class="warning">${message}</div>
+  `);
+};
```

Comment:
successHandler and errorHandler have same logic. it can be one function, which also accept className as argument